build/lib/versions.txt

```diff
@@ -8,13 +8,13 @@ bcpkix-jdk15on.jar                  | 1.53
 bcprov-jdk15on.jar                  | 1.53                                                         | See http://www.bouncycastle.org/licence.html
 cglib.jar                           | 2.1.3 (JMock 2.1.0)                                          |
 commons-lang.jar                    | 2.3                                                          | Apache 2.0
-commons-logging.jar                 | Jetty 5.1.10                                                 | Apache 2.0
```

Comment:
No, it will not (Travis successfully builds the build, which is a good indication that the build isn't broken ;) ).  The API signature provided by jcl-over-slf4j.jar is similar to commons-logging 1.1.1.